imagepicker/src/main/res/values-ru/strings.xml

```diff
@@ -17,5 +17,5 @@
     <string name="ef_msg_no_camera_permission">Дайте разрешение, чтобы сделать снимок</string>
     <string name="ef_msg_limit_images">Лимит выбора изображений</string>
     
-    <string name="ef_image_directory" translatable="false">Камера</string>
+    <string name="ef_image_directory" translatable="false">Camera</string>
```

Comment:
What i mean is, it shouldn't be here in the first place. Please delete it. 

Sorry if i'm not make myself clear on the first. Thanks